google/gax/api_callable.py

```diff
@@ -254,29 +251,55 @@ def _construct_retry(method_config, retry_codes, retry_params, retry_names):
     Returns:
       A RetryOptions object, or None.
     """
-    if not method_config:
+    if method_config is None:
         return None
-    codes = []
-    if retry_codes:
-        for codes_name in retry_codes:
-            if (codes_name == method_config['retry_codes_name'] and
-                    retry_codes[codes_name]):
-                codes = [
-                    retry_names[name] for name in retry_codes[codes_name]]
-                break
 
-    params_struct = None
-    if method_config.get('retry_params_name'):
-        for params_name in retry_params:
-            if params_name == method_config['retry_params_name']:
-                params_struct = retry_params[params_name].copy()
-                break
-        backoff_settings = BackoffSettings(**params_struct)
-    else:
-        backoff_settings = None
+    codes = None
+    if retry_codes and 'retry_codes_name' in method_config:
```

Comment:
Sorry, not related to this change, but could you fix the docs for the `retry_codes` param? It has the wrong name.